src/formatters/inline_text_formatter.js

```diff
@@ -0,0 +1,31 @@
+// Text format for easy machine parsing.
+import columnify from 'columnify';
+
+export default function InlineTextFormatter(errorsGroupedByFile) {
+  const files = Object.keys(errorsGroupedByFile);
+
+  const errorsText = files
+    .map(file => {
+      return generateErrorsForFile(file, errorsGroupedByFile[file]);
+    })
+    .join('\n');
+  return errorsText + '\n';
+}
+
+function generateErrorsForFile(file, errors) {
+  const formattedErrors = errors.map(error => {
+    const location = error.locations[0];
+
+    return {
+      location: `${file}:${location.line}:${location.column}`,
+      message: error.message,
+      rule: ` ${error.ruleName}`,
+    };
+  });
+
+  const errorsText = columnify(formattedErrors, {
```

Comment:
Kinda related to my question about colors, did you intentionally want `columnify` here? It'll basically align each value in the above hash, like this:

<img width="736" alt="screenshot-v0 0 24" src="https://user-images.githubusercontent.com/385270/72854591-ce227080-3c82-11ea-9e61-c49a4572847b.png">


My gut feeling is you could simplify this entire function and do something like this:

```js
function generateErrorsForFile(file, errors) {
  const formattedErrors = errors.map(error => {
    const location = error.locations[0];

    return `${file}:${location.line}:${location.column} ${error.message} ({error.ruleName})`
  });

  return formattedErrors.join("\n");
}
```

Assuming the output you want is:

```
schema.graphql:2:1 The enum `CommentStatus` should be sorted alphabetically. (enum-values-sorted-alphabetically)
schema.graphql:12:1 The enum `SomethingElse` should be sorted alphabetically. (enum-values-sorted-alphabetically)
```

Reply:
Yes its not necessary to have the nice columns. I'm also thinking about the name and I'm not crazy about it, what do you think about 'parsable' or 'compact' instead?